app/src/main/java/com/moxielabs/androidarchitectureexamples/mvvm/viewmodels/MainViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package com.moxielabs.androidarchitectureexamples.mvvm.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.moxielabs.androidarchitectureexamples.shared.models.Fruit
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class MainViewModel : ViewModel() {
```

Comment:
How would you feel about renaming this? Since it isn't tied to MainActivity anymore

Reply:
Excellent idea